src/css/homepage.css

```diff
@@ -0,0 +1,43 @@
+.events-section {
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+  margin: 0;
+}
+
+.event-item {
+  position: relative;
+  width: calc(50% - 40px);
+  min-height: 140px;
+  margin: 16px;
+  padding: 16px;
+  box-shadow: #0000001c 0px 0.3px 0.9px, rgba(0, 0, 0, 0.11) 0px 1.6px 3.6px;
+}
+
+.event-item div {
+  color: #78757f;
+  font-style: italic;
+}
+
+.event-item a {
+  position: absolute;
+  right: 16px;
+  bottom: 16px;
+  background-color: black;
+  color: white;
+  text-decoration: none;
+  padding: 4px 8px;
+  border-radius: 8px;
+}
+
+.event-item a:hover {
+  opacity: 0.8;
+}
+
+#upcoming-events-section .event-item a {
+  background-color: #2a9d8f;
```

Comment:
I ran a [color contrast check](https://webaim.org/resources/contrastchecker/) between this color and the white text, and I'm getting a contrast ratio of **3.32:1**. [Text this size should meet a ratio of **4.5:1**](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html).

We can remedy our contrast two ways:

* We could find a darker, more contrasting background color.
* We could make the link contents "large text," which only needs to meet a **3:1** ratio. Large text can be either:
    * At least `18pt`, or
    * At least `14pt` and bold

I'm happy to leave it up to you to decide how you'd like to go about handling this contrast.